### ISSUE TITLE
ModuleNotFoundError: No module named 'tensorflow.keras.layers.experimental'Bug: 

### ISSUE THREAD

**Issue by veiro** [keras-team/autokeras#1904]
### Bug Description
when import autokeras

import autokeras as ak

### Bug Reproduction
Code for reproducing the bug:

https://colab.research.google.com/github/keras-team/autokeras/blob/master/docs/ipynb/structured_data_classification.ipynb  

### Setup Details
Include the details about the versions of:
 - OS type and version:
 - Python: 3
 - autokeras: 
 - keras-tuner:
 - scikit-learn:
 - numpy:
 - pandas:
 - tensorflow:

### Additional context

[<ipython-input-4-4e35e895c450>](https://localhost:8080/#) in <cell line: 4>()
      2 import tensorflow as tf
      3 
----> 4 import autokeras as ak

7 frames
[/usr/local/lib/python3.10/dist-packages/autokeras/keras_layers.py](https://localhost:8080/#) in <module>
     18 from tensorflow import nest
     19 from tensorflow.keras import layers
---> 20 from tensorflow.keras.layers.experimental import preprocessing
     21 
     22 from autokeras.utils import data_utils

ModuleNotFoundError: No module named 'tensorflow.keras.layers.experimental

**Comment by Aya121298** [comment]
I have the same problem

**Comment by sorowar-cse** [comment]
SaMEEEEEEEE

**Comment by JingxinLee** [comment]
same

**Comment by 2abet** [comment]
The experimental bit has been rendered redundant. Use tf.keras.layers import RandomFlip instead.
https://www.tensorflow.org/api_docs/python/tf/keras/layers/RandomFlip

**Comment by odeyale2016** [comment]
from tensorflow.keras.layers import *
The above code works for me

**Comment by nikitosssobur** [comment]
This problem is still relevant! I have the following version configuration: `python 3.12.8  tensorflow 2.18.0  autokeras 1.0.20`.

**Comment by boguskiP** [comment]
I must agree with nikitosssobur, still an issue with the following configuration: 
`python  3.12.9 
tensorflow 2.18.0 
autokeras 1.0.20`

**Comment by bangtanthorn** [comment]
autokeras worked for me with the versions: 
AutoKeras Version: 1.0.20
TensorFlow Version: 2.12.0
Keras Version: 2.12.0
Python Version: 3.11.12
